doc/conf.py

```diff
@@ -130,9 +130,19 @@
     'within_subsection_order': FileNameSortKey,
     'ignore_pattern': '/generate_',
     'nested_sections': False,
-    'copyfile_regex': r'.*\.rst|.*\.png|.*\.svg'
+    'copyfile_regex': r'.*\.rst|.*\.png|.*\.svg',
+    'filename_pattern': '/plot_',
 }
 
+if tags.has("handle_drift") or tags.has("all_long_plot"):
```

Comment:
Nice! Worth generalising this? Something like:

```suggestion
long_tutorial_names = ["handle_drift"]

for long_tutorial_name in long_tutorial_names:
    if tags.has(long_tutorial_name) or tags.has("all_long_plot"):

        if (tutorial_path := (Path('long_tutorials/'+long_tutorial_name))).is_dir():
            shutil.rmtree(tutorial_path)

        sphinx_gallery_conf['examples_dirs'].append('../examples/long_tutorials/'+long_tutorial_name)
        sphinx_gallery_conf["gallery_dirs"].append(tutorial_path.as_posix())
```

Reply:
I think this is very nice, for now if it works for you I will wait for the upcoming PR tackling the Neuropixels how-to to implement this, just to make checking it's working for multiple cases a bit easier. I'll link to this suggestion in our issue to not forget!